apps/builder/app/shared/copy-paste/init-copy-paste.ts

```diff
@@ -1,25 +1,16 @@
 import { toastError } from "../error/toast-error";
-import { $authTokenPermissions } from "../nano-states";
+import {
+  $authTokenPermissions,
+  $textEditingInstanceSelector,
+} from "../nano-states";
 
 const isTextEditing = (event: ClipboardEvent) => {
-  const selection = document.getSelection();
-
-  if (selection?.type === "Range" || selection?.type === "Caret") {
+  // Text is edited on the Canvas using the default Canvas text editor settings.
+  if ($textEditingInstanceSelector.get() != null) {
```

Comment:
can't be null, lets be explicit if we can

```suggestion
  if ($textEditingInstanceSelector.get() !== undefined) {
```

Reply:
We are gonna use /* eslint eqeqeq: ["error", "always", {"null": "never"}] */ soon.